src/Mono.Android/Mono.Android.csproj

```diff
@@ -25,7 +25,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <IncludeAndroidJavadoc Condition=" '$(IncludeAndroidJavadoc)' == '' And '$(CONVERT_JAVADOC_TO_XMLDOC)' == 'true' And '$(AndroidFrameworkVersion)' == '$(AndroidLatestStableFrameworkVersion)'">True</IncludeAndroidJavadoc>
+    <IncludeAndroidJavadoc Condition=" '$(IncludeAndroidJavadoc)' == '' And '$(CONVERT_JAVADOC_TO_XMLDOC)' == 'true' And ('$(AndroidFrameworkVersion)' == '$(AndroidLatestStableFrameworkVersion)' or '$(TargetFramework)' != 'monoandroid10')">True</IncludeAndroidJavadoc>
```

Comment:
I'm not sure if the `!= monoandroid10` condition is needed here? We should only be building the net6.0 version of Mono.Android.dll when `$(AndroidFrameworkVersion)' == '$(AndroidLatestStableFrameworkVersion)'`. Probably doesn't hurt to have this either way though, it may be useful in the future if/when we produce a Ref pack for a preview API level?

Reply:
I was probably seeing that `$(CONVERT_JAVADOC_TO_XMLDOC)` was blank, and we don't need it *now*.

But since we would need this when we have an API 32 "preview" for `net6.0-android32`, maybe we should leave it?